README.md

```diff
@@ -134,11 +134,39 @@ services:
     - ...
 
 ```
+### Start with an appliance backup
+
+To start with the appliance backup files (See [here](https://github.com/mekomsolutions/appliance-deployment/blob/main/README.md#backup-profile) for more details on how to get the backup files), follow these steps:
+
+1. Unzip the backup file and rename PostgreSQL database files to:
+  - OpenELIS : `clinlims.tar`
+  - Odoo : `odoo.tar`
+2. Move PostgreSQL database files to [./sqls/postgresql/restore](./sqls/postgresql/restore) folder
+3. For OpenMRS database please folow the steps [here](#start-with-a-custom-mysql-dump)
+4. Unzip the `filestore.zip` file and set the variables in `.env` file as following:
+   - Odoo:
+     - ODOO_FILESTORE=`<filestore-path>/odoo`
```

Comment:
> <filestore-path>
What does this refer to? The path where the filestore is after having extracted the ZIP?

Reply:
Yes

Reply:
Then the var name should be more clear or explained above it.